mimesis/utils.py

```diff
@@ -87,7 +87,7 @@ def get_data(locale_name: str) -> JSON:
 
     locale = locale.lower()
 
-    if locale not in config.SUPPORTED_LOCALES:
+    if locale not in locales.SUPPORTED_LOCALES:
```

Comment:
Code duplication. We have this logic in `override_locale` as well. Needs a refactor.